### ISSUE TITLE
Replay TV stops after one episode

### ISSUE THREAD

**Issue by FRL-Myke** [rbuehlma/pvr.zattoo#171]
Not quite sure if it is a bug or a feature but i'll post it here anyway. Might also be i missed a setting.

Anyway, first the used Hardware/Software:
Formuler Z11 Pro Max
Android TV 11
Kodi 20.1
Your addon 20.3.7

Observed Behaviour:
When watching a TV Show in Replay, after the show finished and the next show started, after a few minutes the Playback stops apruptly.

Expected Behaviour:
As sometimes there are multiple episodes of a show broadcasted one after the other, i would expect that the player continues playing even after the initially started episode finished.
The native standalone App from Zattoo works that way, it continues playing until the user stops or switches.

Theory:
As Zattoo "cuts" the programm for replay and adds some buffer before and after the show (because the stations are rarely on time), the addon treats these episodes as standalone video stream and stops once the stream ends.
I'm wild guessing, sorry.

Do i miss a setting here, is it a bug or a feature?

Thank you for your time and sorry for bothering you.

**Comment by rbuehlma** [comment]
It's neither a bug nor a feature. I would also prefer it in your expected behaviour, unfortunately, this is currently not possible with a PVR addon. For this to be possible, the API between Kodi and the PVR addons will need to be extended. I have already created a feature request against Kodi for this but it's unclear when it will be implemented. Also, I can't reference that feature request here until the Kodi forum is back online. Closing this issue since it it planned to be implemented as soon as the API allows it.
